tests/bench/large.js

```diff
@@ -50947,7 +50947,7 @@ exports.debuglog = function(set) {
 
 
 /**
- * Echos the value of a value. Trys to print the value out
+ * Echos the value of a value. Tries to print the value out
```

Comment:
it is just a copy of jshint. I don't think we want to maintain it. :)

Reply:
I've rebased this out